axlearn/audio/asr_decoder.py

```diff
@@ -28,6 +28,40 @@
 from axlearn.common.utils import Nested, Tensor
 
 
+def _is_valid_ctc_seq(paddings: Tensor, target_labels: Tensor, target_paddings: Tensor) -> Tensor:
+    """Returns for per example sequence if it passes validity check.
```

Comment:
```suggestion
def _is_valid_ctc_seq(*, paddings: Tensor, target_labels: Tensor, target_paddings: Tensor) -> Tensor:
    """Returns whether each input sequence passes validity check.
```

Reply:
Are the changes pushed?

Reply:
thanks for catching this. just pushed another one to fix it.